src/Microsoft.PowerShell.Commands.Utility/commands/utility/NewGuidCommand.cs

```diff
@@ -0,0 +1,49 @@
+
+/********************************************************************++
+Copyright (c) Microsoft Corporation.  All rights reserved.
+--********************************************************************/
+
+using System;
+using System.Management.Automation;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// The implementation of the "new-guid" cmdlet
+    /// </summary>
+    ///
+    [Cmdlet(VerbsCommon.New, "Guid", HelpUri = "https://go.microsoft.com/fwlink/?LinkId=526920", DefaultParameterSetName = "Count")]
+    [OutputType(typeof(Guid))]
+    public class NewGuidCommand : Cmdlet
+    {
+        /// <summary>
+        /// True to return the Guid.Empty
+        /// </summary>
+        [Parameter(ParameterSetName = "Empty")]
+        public SwitchParameter Empty { get; set; }
+
+        /// <summary>
+        /// The number of guids to generate
+        /// </summary>
+        [Parameter(ParameterSetName = "Count")]
```

Comment:
Well, to me, that looks like the input in some way should be related to the created guids.